### ISSUE TITLE
bug parsing 1-byte logical columns

### ISSUE THREAD

**Issue by sbailey** [esheldon/fitsio#59]
An example Tractor catalog file from the DECam Legacy Survey is available here:

http://portal.nersc.gov/project/cosmo/data/legacysurvey/dr1/tractor/000/tractor-0001m002.fits

It contains a binary table that has some TFORMnn = 'L' columns for 1-byte LOGICAL, e.g. the column 'brick_primary'.  These appear to be mis-parsed by fitsio, resulting in all False values.  astropy.io.fits has many of these as True.

```
import fitsio
from astropy.io import fits

infile = "/project/projectdirs/cosmo/data/legacysurvey/dr1/tractor/000/tractor-0001p027.fits"
d1 = fitsio.read(infile)
d2 = fits.getdata(infile)

len(d1), np.count_nonzero(d1['brick_primary'])  #- (4984, 0)
len(d2), np.count_nonzero(d2['brick_primary'])  #- (4984, 4548)
```

**Comment by sbailey** [comment]
Forgot to mention: problem originally found on fitsio 0.9.5, but persists after upgrading to fitsio 0.9.7.

**Comment by esheldon** [comment]
Can you please try installing the release candidate for 0.9.8?  There
were some fixes for this sort of column.

On 10/9/15, sbailey <notifications@github.com> wrote:
> Forgot to mention: problem originally found on fitsio 0.9.5, but persists
> after upgrading to fitsio 0.9.7.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/esheldon/fitsio/issues/59#issuecomment-146946808


-- 
Erin Scott Sheldon
Brookhaven National Laboratory erin dot sheldon at gmail dot com

**Comment by sbailey** [comment]
This is already fixed in 0.9.8rc1 .  Thanks.

Note that the setup.py file of 0.9.8rc1 still stays version="0.9.7".
